ipfs-paste

```diff
@@ -64,25 +73,38 @@ fi
 if [ $paste ]; then
   test -n "$clipaste" || die "clipboard paste unkown for $(uname)"
   log "paste to ipfs..."
-  fhash=$($clipaste | ipfs add -q)
+
+  date=$(date +"%Y-%m-%dT%H:%M:%SZ")
+  fpath=$(mktemp "/tmp/paste.$date.XXXXXX.txt") ||
+  die "could not 'ktemp /tmp/paste.$date.XXXXXX.txt'"
```

Comment:
Yeah, I understand that it can be confusing for the reader. I found it confusing myself when I started working on Git.

I asked on the Git mailing list:

http://thread.gmane.org/gmane.comp.version-control.git/274665